execution/executor/src/block_processor.rs

```diff
@@ -633,8 +636,18 @@ where
                 num_accounts_created,
             ));
             current_state_tree = state_tree;
+
+            // check for change in validator set
+            let validator_set_change_event_key = ValidatorSet::change_event_key();
+            for event in vm_output.events() {
+                if *event.key() == validator_set_change_event_key {
+                    next_validator_set = Some(ValidatorSet::from_bytes(event.event_data())?);
+                    break;
+                }
+            }
         }
 
+<<<<<<< HEAD
```

Comment:
merge conflict lol

Reply:
Oops, forgot to push my local fix for that...